README.md

```diff
@@ -1 +1,167 @@
-# Capstone-1
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+
+  <h3><b>CAPSTONE 1 - COMMUNITY SPARK</b></h3>
+
+</div>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+# 📖 Capstone 1 <a name="about-project"></a>
+
+**Capstone 1** is part of introductory projects in Microverse software development program.
+The template adapted for the project is for an event program. Some of its contents were redacted as part of the provided guidlines.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+- HTML
+- CSS
+- Javascript
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="">N/A</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="">N/A</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="">N/A</a></li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- **Build an event website**
+- **Dynamic loading of website contents**
+- **Click controls of website elements**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link - N/A]()
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+In order to run this project you need Live Server for your local testing
+
+### Setup
+
+Clone this repository to your desired folder on your local machine
+
+### Install
+
+Install this project with:
+
+### Usage
+
+To run the project, execute the following command:
+
+N/A
```

Comment:
- [ ] Good job adding a more clear README file, however, you should provide information about how to use your web page, if a user what to run linters for example, or clone the project, :smile:, what do you think about completing your README file ? Take a look at this one [here](https://github.com/iambenkis/Global-summit-GMIS-/blob/feature-branch/README.md) :eyes: